keep-ui/features/workflows/builder/ui/Editor/TriggerEditor.tsx

```diff
@@ -33,9 +44,20 @@ export function TriggerEditor() {
 
   const updateAlertFilter = (filter: string, value: string) => {
     const currentFilters = properties.alert || {};
-    const updatedFilters = { ...currentFilters, [filter]: value };
-    updateV2Properties({ alert: updatedFilters });
-    saveNodeDataDebounced("properties", { alert: updatedFilters });
+    if (!currentFilters.filters) {
+      currentFilters.filters = {};
+    }
+    currentFilters.filters[filter] = value;
+    updateV2Properties({ alert: currentFilters });
+    saveNodeDataDebounced("properties", { alert: currentFilters });
```

Comment:
it's confusing, but to correctly display the trigger node `saveNodeDataDebounced` should receive the properties not wrapped in {alert: ...}, because it's different from workflow properties.

Also, it's better to create new object to avoid possible state inconsistency.
```suggestion
    saveNodeDataDebounced("properties", {...currentFilters, [filter]: value});
```